src/constants/db.ts

```diff
@@ -48,7 +48,7 @@ export function generateBaseDBfields() {
 }
 
 export interface AccessSchema extends IdAndSchemaVersion {
-	access_level: Access;
```

Comment:
There is a mismatch between the property name in typescript and the name saved on the database.